Tella/Data/Networking/Repositories/GDriveRepository.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  GDriveRepository.swift
+//  Tella
+//
+//  Created by gus valbuena on 5/28/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import GoogleSignIn
+import GoogleAPIClientForREST
+import Combine
+
+protocol GDriveRepositoryProtocol {
+    func handleSignIn() async throws -> Void
+    func restorePreviousSignIn() async throws -> Void
+    func handleUrl(url: URL)
```

Comment:
maybe use directly : 
```
func handleSignIn() async throws
func restorePreviousSignIn() async throws 
```

Reply:
this is done in this [pr](https://github.com/Horizontal-org/Tella-iOS/pull/147)